tests/evaluator/haaqi/test_haaqi.py

```diff
@@ -16,4 +16,4 @@ def test_haaqi_v1() -> None:
     level1 = 65
 
     score, _, _, _ = haaqi_v1(x, sr, y, sr, hl, eq, level1)
-    assert score == pytest.approx(0.1090639807843874, rel=1e-2)
+    assert score == pytest.approx(0.109534910970557, rel=1e-6)
```

Comment:
Don't have a strong opinion but this is `1e-6` and the `test_hasqi.py` is `1e-7`.

Reply:
Yeah, still trying to see why I get different results locally vs github
1- The linear algebra library that numpy uses?
2- CPU?